app/src/main/java/com/github/se/polyfit/ui/screen/OverviewScreen.kt

```diff
@@ -134,14 +152,11 @@ fun OverviewScreen(
             listOf(
                 overviewViewModel.callCamera(context, startCamera, requestPermissionLauncher),
                 {
-                  Toast.makeText(
-                          context, context.getString(R.string.PermissionDenied), Toast.LENGTH_SHORT)
-                      .show()
+                  pickMedia.launch(
+                      PickVisualMediaRequest(ActivityResultContracts.PickVisualMedia.ImageOnly))
                 },
                 {
-                  Toast.makeText(
-                          context, context.getString(R.string.PermissionDenied), Toast.LENGTH_SHORT)
-                      .show()
+                  Toast.makeText(context, "Feature not yet implemented", Toast.LENGTH_SHORT).show()
```

Comment:
The old one is better, all the shown text is in the string.xml file

Reply:
How is the message @antomax33  ? The text will be moved to the string.xml`